ccd-definition/AuthorisationCaseField/CareSupervision/solicitor.json

```diff
@@ -350,6 +402,175 @@
     ],
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "groundsForEPO",
```

Comment:
For this one, I tried restricting the access to only R and I get the 'No field found' error, looks like we have to give full permissions to all the fields from the 'enterGrounds.json' file to avoid this, which is why I have given permissions to some fields that are unrelated to this application